cfy_manager/components/prometheus/config/prometheus.yml

```diff
@@ -0,0 +1,55 @@
+# Prometheus self-check
+global:
+  scrape_interval:     15s
+  evaluation_interval: 30s
+  # scrape_timeout is set to the global default (10s).
+
+  external_labels:
+    monitor: cloudify
+
+# Alertmanager configuration
+#alerting:
+#  alertmanagers:
+#    - static_configs:
+#        - targets:
+#          # - alertmanager:9093
```

Comment:
@mateumann are we going to keep these commented ? or we are override them during the code ?

Reply:
no, I removed those lines in the last commit